src/stats.rs

```diff
@@ -0,0 +1,34 @@
+use std::borrow::Borrow;
+use api::{Id, Collection, Bool, OwnerId};
+
+#[cfg(feature = "unstable")]
+include!("stats.rs.in");
+
+#[cfg(not(feature = "unstable"))]
+include!(concat!(env!("OUT_DIR"), "/stats.rs"));
+
+request_ref! {
+    #[derive(Copy, Eq)]
+    struct Get for ["stats.get"](v => 5.44) -> Collection<Period> {
+        sized {
+            group_id: Option<Id> = () => {Option},
+            app_id: Option<Id> = () => {Option},
+        }
+        unsized {
+            date_from: str = ("") => {=},
+            date_to: str = ("") => {=},
```

Comment:
I know the type of `date_from` and `date_to` fields are "string" according to docs, but it also says "Latest datestamp (in Unix time)", which implies it's an integer (presumably `u64` a.k.a. `Timestamp` in this case).